examples/upgradeable/README.md

```diff
@@ -0,0 +1,19 @@
+# Hardhat Sample for Hardhat Ignition
+
+This project is Hardhat's sample project enhanced with Hardhat Ignition.
```

Comment:
We should update this intro to explain that we are showing upgradeable contracts.

Reply:
Oh whoops! good catch